.github/workflows/build.yaml

```diff
@@ -472,3 +511,23 @@ jobs:
             tag=${{ env.BRANCH }}
             dockerRepository=${{ env.REGISTRY }}
             platform=x86_64
+
+  trigger-workflow:
+    name: Trigger workflow depending upon the branch
+    needs: [build-obr, build-obr-generic, build-obr-javadocs]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Triggering helm build (github.ref is main)
+        if: env.BRANCH == 'main'

```

Comment:
Does this need to be `${{ env.BRANCH }}`?

Reply:
done